benchmarks/src/test/java/io/crate/execution/engine/collect/collectors/OrderedLuceneBatchIteratorBenchmark.java

```diff
@@ -62,12 +67,18 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
 
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.SECONDS)
 @State(Scope.Benchmark)
 public class OrderedLuceneBatchIteratorBenchmark {
 
+    private static final RamAccountingContext RAM_ACCOUNTING_CONTEXT =
+        new RamAccountingContext("dummy", new NoopCircuitBreaker(CircuitBreaker.FIELDDATA));
+    private static final Supplier<RowAccounting> ROW_ACCOUNTING_SUPPLIER = () ->
+        new RowAccounting(Collections.singleton(LongType.INSTANCE), RAM_ACCOUNTING_CONTEXT);
```

Comment:
Minor: you could do the new RamAccountingContext("dummy"...) inline here since it's not used independenly